### PR TITLE
cni: Switch CNI to Cilium from Weave Net

### PR DESCRIPTION
**Pull request by klausenbusk**
Cilium[1] offers more features and seems to be what most people are
using (so easier to get help).

[1] https://cilium.io/

**Description of your changes:**

Checklist:

* [x] I have bumped the chart version
* [x] I have documented my changes if this is needed
* [x] I have described my changes in the "description of your changes" field
* [x] I have tested this change on a running cluster and the Argocd dashboard shows healthy and synced
   * It has been tested without Argo, but traffic is flowing.
* [ ] I have created the necessary tests in the chart, if they are possible/necessary
* [x] I have squashed commits if necessary

### CODE REVIEW

File: charts/cni/chart/Chart.yaml [hunk 1]
```diff
@@ -1,4 +1,9 @@
 apiVersion: v2
 name: cni
 description: A helm chart for our CNI
-version: 0.3.5
+version: 0.3.6
```
Comment (jaxels10) [inline charts/cni/chart/Chart.yaml:4]: Should we consider bumping to 1.0.0 since we are now changing the type of CNI?

**Review by jaxels10 — COMMENTED** [review]
A single comment

File: charts/cni/chart/Chart.yaml (hunk 1 above)
Comment (klausenbusk) [reply charts/cni/chart/Chart.yaml]: SGTM.

**Review by jaxels10 — APPROVED** [review]
This LGTM

**Pull request merged**